newIDE/app/src/UI/Theme/DefaultTheme/index.js

```diff
@@ -13,8 +17,20 @@ const systemSelectionColor = '#4c92ff'; //OS X selection
 
 // Use the fonts provided by the operating system(s) as possible.
 // If you update this font list, be sure to do it in all the other places using fonts in the codebase.
-const fontFamily =
-  "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, 'Helvetica Neue', Helvetica, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'";
+const fontFamily = CSS.list(
```

Comment:
I would rather not have this change in this PR, because if we change one list like this we should do it for all other similar lists in the codebase for consistency.